bids/cli.py

```diff
@@ -0,0 +1,123 @@
+from pathlib import Path
+import click
+
+from . import __version__
+
+# alias -h to trigger help message
+CONTEXT_SETTINGS = {'help_option_names': ['-h', '--help']}
+
+
+class PathOrRegex(click.ParamType):
+    "A helper Type to parse BIDSLayoutIndexer ignore/force entries"
+    name = "path or /regex/"
+
+    def convert(self, value, param, ctx):
+        if value.startswith('/') and value.endswith('/'):
```

Comment:
this is tricky, since there are a lot of patterns that could compile to match everything. I don't know what the best way around this, besides letting the users see what works for themselves.

i'm hesitant to do this, but we could add a `--debug` option to show the user all passed in options / compiled regexes.